packages/volto/src/components/manage/Edit/Edit.jsx

```diff
@@ -286,6 +286,7 @@ class Edit extends Component {
       <Form
         isEditForm
         ref={this.form}
+        navRoot={this.props.content?.['@components']?.navroot?.navroot || {}}
```

Comment:
This is not consistent with the Add form, which uses `this.props.content?.['@components']?.navroot`

Reply:
Fixed in https://github.com/plone/volto/pull/5424/commits/24589eb3e89a5f6f885b4e04dda526c1e3533995